lib/spark_api/models/listing_cart.rb

```diff
@@ -52,6 +63,10 @@ def self.portal(arguments={})
           collect(connection.get("/#{self.element_name}/portal", arguments))
       end
 
+      def deletable?
+        !attributes.has_key?("PortalCartType")
```

Comment:
I'm not positive this logic is correct. This should be true for custom carts, and false for vow carts like "Favorites". Custom carts don't appear to have this attribute, but I would be more comfortable if the attribute were more closely matched to the purpose.